src/widget/wtracktableview.cpp

```diff
@@ -852,6 +855,24 @@ void WTrackTableView::slotAddToAutoDJReplace() {
     addToAutoDJ(PlaylistDAO::AutoDJSendLoc::REPLACE);
 }
 
+void WTrackTableView::slotCurrentColumnChanged() {
+	QAbstractItemModel* itemModel = model();	
+    TrackModel* trackModel = dynamic_cast<TrackModel*>(itemModel);
+   
+    this->selectionModel()->setCurrentIndex(this->model()->index(currentIndex().row(), trackModel->columnIndexFromSortColumnId(static_cast<TrackModel::SortColumnId>(static_cast<int>(m_pCurrentColumn->get())))), QItemSelectionModel::Current);
```

Comment:
We don't use this-> in c++.
Please break long lines. 
I suggest to use the pre-commit hook it will take care for that automatically.

Reply:
https://github.com/mixxxdj/mixxx/wiki/Using-Git#set-up-automatic-code-checking

Reply:
Thanks for the hint, but I'm on Windows and this is only available on Linux.
But I will fix his of cause!

Reply:
I'm pretty sure it's also available on windows if you install clang-format manually.